app/routes/_layout.clusters.$clusterName.($filterNamespace).chart-releases.add.$chartName.tsx

```diff
@@ -105,14 +105,14 @@ export async function action({ request, params }: ActionArgs) {
     cluster: params.clusterName,
```

Comment:
I think you'll need to do a `helmfileRefEnabled: formData.get("helmfileRefEnabled") === "true"` here and in the environments add chart name equivalent but I could be wrong -- I think you'll hit a type error if you don't

Reply:
indeed! It showed up in my manual testing. On it! thanks :)